### DISCUSSION TITLE
Central Database

### DISCUSSION THREAD

**Discussion by GameTec-live** [ThioJoe/YT-Spammer-Purge#168] [Ideas]
Why not create a Central database with known spam accounts? So the programm has an option to delte them from the database and maybe a option to add the newly found accounts to the database. Id help (know a good amount of python)... Just want to know if its a good or just stupid idea...

**Comment by TechStudent10** [comment]
Pretty sure someone else posted something like this.

**Comment by ThioJoe** [comment]
I thought about it, but there are a couple problems that I think make it not very practical. 

First, these spammers are constantly making new accounts, so banning one wouldn't do much, they won't be using it for very long. 

Second, if everyone is basically using the same filtering mode to find the scammers, there's not really a point in also searching a database if each comment has to be scanned anyway for the channel ID, and then you'd also be scanning with the filter, and detect them just like the original person did.

**Comment by GameTec-live** [comment]
Well, i guess but i more thought about banning them before they leave a comment. Some kind of prevention....

**Comment by rbeer** [comment]
You will need someone that invests a good amount of time sanity checking that database. Otherwise, you will end up with a lot of false positives being banned across the platform. Not to mention deliberate attacks, adding legitimate people.

This is not so much a short-term problem of implementing the feature, but rather a long-term one of making sure it's not abused.

**Reply by TechStudent10** [reply]
totally

**Reply by GameTec-live** [reply]
well ig thats true...
